presto-hive/src/test/java/com/facebook/presto/hive/TestHiveRecoverableExecution.java

```diff
@@ -459,6 +460,7 @@ private static Session createRecoverableSession(int writerConcurrency, boolean o
                 .setSystemProperty(MAX_STAGE_RETRIES, "4")
                 .setCatalogSessionProperty(HIVE_CATALOG, VIRTUAL_BUCKET_COUNT, "16")
                 .setCatalogSessionProperty(HIVE_CATALOG, OPTIMIZED_PARTITION_UPDATE_SERIALIZATION_ENABLED, optimizedPartitionUpdateSerializationEnabled + "")
+                .setCatalogSessionProperty(HIVE_CATALOG, ENABLE_REPLICATED_READS_FOR_BROADCAST_JOIN, "false") // replicated reads is not supported for grouped execution so won't apply for recoverable GE
```

Comment:
The grouped execution cannot be applied with replicated reads at this phase of development because the local execution for the `PerBucket` implementation maintains one AsyncQueue for each bucket. Replicated reads strategy cannot create multiple AsyncQueue tasks for all buckets which could potential cause `HIVE_EXCEEDED_SPLIT_BUFFERING_LIMIT` issue